docs/pf-keys/python/run_script.rst

```diff
@@ -0,0 +1,280 @@
+********************************************************************************
+Run script
+********************************************************************************
+
+================================================================================
+Anatomy
+================================================================================
+
+At the top of all the Python test scripts (located in `parflow/test/python/
+<https://github.com/parflow/parflow/tree/master/test/python>`_) are something similar to the following lines:
+
+.. code-block:: python3
+
+    from parflow import Run
+    test_run = Run("test_run", __file__)
+
+These lines import the ``Run`` class from the ``parflow`` module and create a new ``Run`` object called ``test_run`` (or whatever you want the run name to be). All the key/value pairs are set on this object.
```

Comment:
What is "__file__" argument for?

Reply:
This is to attach the working directory to the one that contains that file. Otherwise it would be where ever you run your python command line.